tests/service/test_interface.py

```diff
@@ -273,3 +274,11 @@ def test_get_task_by_id(context_mock: MagicMock):
 def test_stomp_config():
     interface.set_config(ApplicationConfig(stomp=StompConfig()))
     assert interface.messaging_template() is not None
+
+
+@pytest.mark.stomp
+def test_stomp_connection_failure_with_invalid_config(caplog):
+    caplog.at_level(logging.exception)
+    interface.set_config(ApplicationConfig(stomp=StompConfig(host="Not Exists")))
```

Comment:
please do not do 3 nested inits in one line

Reply:
ruff prefers it on one line